lib/layout/adaptive.dart

```diff
@@ -3,12 +3,14 @@
 // found in the LICENSE file.
 
 import 'package:adaptive_breakpoints/adaptive_breakpoints.dart';
+import 'package:dual_screen/dual_screen.dart';
 import 'package:flutter/material.dart';
 
 /// Returns a boolean value whether the window is considered medium or large size.
 ///
 /// Used to build adaptive and responsive layouts.
 bool isDisplayDesktop(BuildContext context) =>
+    !isDisplayFoldable(context) &&
```

Comment:
Tried to remove the check but the results are worse, I would say. The main reason is that isDisplayDesktop looks at the size of the display and almost all widgets that call the method make the assumption that that they are occupying the whole screen. This is no longer true for the foldable layout, since they occupy 50% of the screen in that scenario.

So to answer your initial question, for the way this app was built (components look at the screen size and not the available size or their own layout), we either have a desktop or a foldable and if the device is an actual foldable desktop - then we treat it as a foldable and not a desktop. Hope this makes sense.

Reply:
Yeah, thanks. Please  update the comment to explain this.